corona-warn-app/cwa-app-android
# Risk tracing stopped correct info (EXPOSUREAPP-4414)

✅   Correct info text (same as for failed one, "Infektionsrisiko")
✅   No Investigation period section ("Ermittlungszeitraum")

![Screenshot 2021-01-28 at 19 20 14](https://user-images.githubusercontent.com/64849422/106168385-44638800-619f-11eb-86ac-9a4233c7ea7e.png)


How to test:
1) start the app
2) turn off tracing
3) go to risk details
4) check correct text and no investigation period section ("Ermittlungszeitraum")
5) turn on tracing
6) if low or high risk - there should be another info text on the bottom and investigation period section in place
